plasmapy/mathematics/mathematics.py

```diff
@@ -1,17 +1,16 @@
 """Functions related to the plasma dispersion function"""
 
 import numpy as np
-from scipy import special
 from astropy import units as u
 from mpmath import polylog
 from scipy.special import wofz as Faddeeva_function
+from typing import Union
 
-# TODO: Add type hint annotations to this module.
 
-
-def plasma_dispersion_func(zeta):
+def plasma_dispersion_func(zeta: Union[complex, int, float, np.ndarray, u.Quantity]
+                           ) -> Union[complex, float, u.Quantity]:
```

Comment:
I think we're missing `np.ndarray` on the return end here. Seems to me like if you pass an array in, an array should come out.

Reply:
Good catch!  It's turning out to be pretty easy to miss type hints...